server/config/config.go

```diff
@@ -193,6 +193,9 @@ type ServerConfig struct {
 	// consider running defrag during bootstrap. Needs to be set to non-zero value to take effect.
 	ExperimentalBootstrapDefragThresholdMegabytes uint `json:"experimental-bootstrap-defrag-threshold-megabytes"`
 
+	// UnsafeAllowClusterVersionDowngrade enables only one single minor cluster version downgrade at a time automatically.
```

Comment:
Let's avoid the term automatic here. It's not really automated in a sense that the operator still needs to babysit rolling upgrade, one by one.

Reply:
Got it. I assumed operator will follow the etcd update practice...